Automattic/wp-calypso
# Site Profiler: Update MetricsSection to new layout

## Proposed Changes

Update the MetricsSection component to new layout.
Add the domain section to the Results Page.

## Why are these changes being made?

To match the Site Profiler V2's layout,

## Testing Instructions

* Go to `/site-profiler/:url`. Ex: `/site-profiler/example.com`
* Check if the image matches the [figma layout](https://www.figma.com/design/P7Jjk4nbUTahUk8aHkG6NQ/Performance-Tool?node-id=712%3A25795&t=cGUjikUMogRe2gO2-1) and the image below:

![CleanShot 2024-05-20 at 15 09 59@2x](https://github.com/Automattic/wp-calypso/assets/5039531/d1a96a63-c3b8-4a1b-a246-05b5de06a247)


## Pre-merge Checklist
